src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -55,13 +55,15 @@
     <description>Used to import the OSCAL profile representing the system's control baseline.</description>
     <define-flag name="href" as-type="uri-reference" required="yes">
       <formal-name>Profile Reference</formal-name>
-      <description>A resolvable URL reference to the profile to use as the system's control baseline.</description>
+      <description>A resolvable URL reference to the profile or catalog to use as the system's control baseline.</description>
```

Comment:
So what's the long-term plan? The element name `import-profile` is misleading still no? Are we going to accept this long-term as the state of affairs to not make a backwards-breaking change or is this a short-term fix or should we place an issue 2.0-ish to make it just `import` or something less counterintuitive?

I am fine approving I guess, just want to understand where we want to go with this given the issue, its ACs, and commentary from everyone involved.

Reply:
We can't change `import-profile` in the short run (re: in a minor or patch release), since this is a backwards compatibility breaking change. We can do this in a major version release thought (i.e., OSCAL v2).

IMHO, we should consider naming this `import-baseline` in OSCAL v2. Would you please create an issue for this?